packages/block-editor/src/components/block-list/block.js

```diff
@@ -380,10 +380,12 @@ function BlockListBlock( {
 		}
 	};
 
-	const onMouseLeave = ( event ) => {
-		// The primary button must be pressed to initiate selection.
+	const onMouseLeave = ( { which, buttons = which } ) => {
```

Comment:
So I haven't tested to check whether it creates an issue or not but if I understand properly the mozilla docs, the `buttons` property could always return `0` in MacOS which means no fallback to which?

Reply:
Hm, that's true. Do you think we should do `buttons || which` instead? So if `buttons` says no button is pressed, we check `which` as well.

Reply:
It seems more logical.

Reply:
Done :)